frontend/src/views/RouteFormView.vue

```diff
@@ -95,7 +95,10 @@
                   <label>{{ $t('routes.starting_mileage') }}</label>
                   <input
                     min="0"
-                    max="1500000"
+                    onkeypress="return event.key === 'Enter'
+                        || (Number(event.key) >= 0
+                        && Number(event.key) <= 9
```

Comment:
What do you think about using `!isNaN(event.target.value)` instead of two conditions with `Number(event.key)` ?

Reply:
Unfortunately that still evaluates to number if characters like `e`, `.`, or `+` are passed